docs/resources/slo.md

```diff
@@ -852,6 +865,17 @@ Optional:
 - `uql` (String) UQL query
 
 
+<a id="nestedblock--objective--count_metrics--bad--logic_monitor"></a>
+### Nested Schema for `objective.count_metrics.bad.logic_monitor`
+
+Required:
+
+- `device_data_source_instance_id` (Number) ID of the metrics stream
+- `graph_id` (Number) Type of data to filter by
+- `line` (String) UQL query
+- `query_type` (String) Query type:
+
```

Comment:
Similar to https://github.com/nobl9/terraform-provider-nobl9/pull/232/files#r1606488476